nvflare/fuel/flare_api/flare_api.py

```diff
@@ -470,7 +470,7 @@ def restart(self, target_type: str, client_names: Optional[List[str]] = None) ->
         if target_type not in _VALID_TARGET_TYPES:
             raise ValueError(f"invalid target_type {target_type} - must be in {_VALID_TARGET_TYPES}")
 
-        parts = [AdminCommandNames.CHECK_STATUS, target_type]
+        parts = [AdminCommandNames.RESTART, target_type]
```

Comment:
This line LGTM.

Can you help just correct the docstring style to follow other methods? (basically the method description should follow """ directly.) thanks

Reply:
I looked and realized there's a bunch of places docstrings can use cleaning up in this file. Is it okay if I put this change with the other changes in a separate PR tomorrow? Or I suppose if this is still open I can add it here.